arm:
  kinematics_solver: fetch_arm_kinematics/IKFastKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
arm_with_torso:
  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3

max_acc_scale: 0.7
max_vel_scale: 0.7

robot_grippers:
  actuator_groups: ['left gripper', 'right gripper']
  gripper_types: ['','suction gripper'] # empty string means its the default claw gripper